src/hearings/services/locations-data.service.ts

```diff
@@ -7,7 +7,10 @@ import { LocationByEPIMMSModel } from '../models/location.model';
 export class LocationsDataService {
   public constructor(private readonly http: HttpClient) {}
 
-  public getLocationById(locationIds: string): Observable<LocationByEPIMMSModel[]> {
-    return this.http.get<LocationByEPIMMSModel[]>(`api/prd/location/getLocationById?epimms_id=${locationIds}`);
+  public getLocationById(locationIds: string, serviceCode: string): Observable<LocationByEPIMMSModel[]> {
+    const options = {
```

Comment:
This is a bit inconsistent. You are using params in options, and also an explicit ?epimms_id=foo in the url. Can use a single approach - probably easiest just to add&serviceCode=${serviceCode} to the end of the url

Reply:
I see your point, will change